src/main/java/carpet/script/api/Inventories.java

```diff
@@ -53,7 +53,13 @@ public static void apply(Expression expression)
         expression.addLazyFunction("item_category", 1, (c, t, lv) ->
         {
             ItemStackArgument item = NBTSerializableValue.parseItem(lv.get(0).evalValue(c).getString());
-            Value res = new StringValue(item.getItem().getGroup().getName());
+            Value res;
+            try {
+                res = new StringValue(item.getItem().getGroup().getName());
+            } catch (NullPointerException n) {
```

Comment:
catching exceptions on a regular is a baaad idea. Check if the group is null and return Value.NULL then.